projects/plugins/jetpack/extensions/blocks/premium-content/_inc/remove-block-keep-content.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { PluginBlockSettingsMenuItem } from '@wordpress/edit-post';
+import { select, dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { transformToCoreGroup } from './transform-to-core-group';
+import { name } from '../index';
+
+function replaceBlockAndKeepContent() {
+	const block = select( 'core/editor' ).getSelectedBlock();
+	dispatch( 'core/block-editor' ).replaceBlock(
+		block.clientId,
+		transformToCoreGroup( block.innerBlocks )
+	);
+}
+
+/**
+ * Register a Menu item for Removing the block and keeping the content.
+ *
+ * @returns {Element} Remove block and keep content menu item.
+ */
+export default () => (
+	<PluginBlockSettingsMenuItem
+		allowedBlocks={ [ name ] }
+		label="Remove block and keep content"
```

Comment:
I could see "Remove block and keep content" seeming a bit contradictory for the average user, but I can't think of anything much more concise than that ("Change premium content to free content" or "Remove content restriction"  or something). 

I don't have a full grasp on block development, but does this section need to be flagged for translation as well?

Reply:
Yeah I think it's fine for now. We'll make sure to flag JPOP Quill to update docs and if we get any feedback, could always update the copy!